addon/store.js

```diff
@@ -131,13 +123,41 @@ var Store = Ember.Object.extend({
 
             var attr = params[0];
             var value = options[attr];
-            var func = function(m) {
+
+            return this._findWithFilterFunc(type, function filterFunction(m) {
                 return get(m, attr) === value;
-            };
-            return this._findWithFilterFunc(type, func);
+            });
         }
+
         return this._findByIdComputed(type, options);
     },
+    findOne(type) {
+        var store = this;
+
+        return Ember.ObjectProxy.extend({
```

Comment:
I would keep unsubscribed undocumented or marked as private for now until we solidify the direction.  Or we can stick with unsubscribe and aim to make all these API changes as part of 5.x